maintenance_equipment_sequence/models/maintenance.py

```diff
@@ -74,7 +74,7 @@ def create(self, vals_list):
                     self.env["ir.sequence"].browse(vals["sequence_id"]).prefix
                 )
         result = super().create(vals_list)
-        result._compute_equipment_code()
```

Comment:
You are reverting in the migration commit a change that was done on purpose in a previous commit in 16.0 https://github.com/OCA/maintenance/pull/455/commits/8972c2828bc7046e8ca8f25a979d772c880ceb30